### DISCUSSION TITLE
Daily High/Low temperatures and summaries

### DISCUSSION THREAD

**Discussion by cloneofghosts** [Pirate-Weather/pirateweather#239] [General]
Continued discussion from #233. Copied and pasted my comment from the PR below

Yeah, this kinda stuff is a bit tricky to figure out. The issue with using 4am to 4am for daily high/low temperatures is that you may miss the highest or lowest temperature of the day if they occur say at 5pm or 6am. Maybe that's why they use 6am to 6am for their daily high/low temperatures?

I'm not fully sure what other APIs do for precipitation but maybe they also consider it between 6am to 6am (or 6am to 6pm and 6pm to 6am for those with day and night forecasts)?

**Comment by alexander0042** [comment]
Exactly! I think it's best to just bit the bullet and use 4-4 for icon/ summary and 6-6 for high/low. It's not that much extra processing, and really would be much clearer

**Comment by cloneofghosts** [comment]
So basically everything is saying the same besides some fixes to the summary/icon as outlined in the PR and in #236?

**Comment by alexander0042** [comment]
Pretty much! Need to clarify that icon is 4am and the others are 6am, but that's it. From a coding side it's a bit more since I have to add another case in (I was using 12am-12am before for the icon/summary by acccudent), but not too bad

**Comment by cloneofghosts** [comment]
Took a quick look at the docs and the temperature high/low already state that they are calculated from 6am to 6am and max/min are 12am to 12am.

The summary and icon don't state the time period in which they were calculated so it's just updating the docs in that case.
